src/DatePickerInput.js

```diff
@@ -41,9 +42,20 @@ const DatePickerInput = React.forwardRef(
       if (formatInputText()) return formatInputText();
       return isDayRange ? getSelectedRangeValue() : getSelectedDayValue();
     };
+    const getSelectedRangeAvailableDays = () => {
+      if (!selectedDayRange.from || !selectedDayRange.to) return '[]';
```

Comment:
Are those single quotes mark around the empty array intentional?

Reply:
Yeah, actually I use `JSON.stringify` to maintain array of days as value of hidden input. Maybe it should be better to find another way